src/views/Account/V4Deposits/BalanceDelegatedTo.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import FeatherIcon from 'feather-icons-react'
+import { formatUnits } from '@ethersproject/units'
+import { useTranslation } from 'react-i18next'
+import { Tooltip, CountUp } from '@pooltogether/react-components'
+import { numberWithCommas } from '@pooltogether/utilities'
+
+import { useUsersAddress } from '@hooks/useUsersAddress'
+import { useUsersTicketDelegateAllPools } from '@hooks/v4/PrizePool/useUsersTicketDelegateAllPools'
+import { useUsersTotalTwab } from '@hooks/v4/PrizePool/useUsersTotalTwab'
+
+export const BalanceDelegatedTo = () => {
+  const usersAddress = useUsersAddress()
+  const { t } = useTranslation()
+
+  const { data: twabs, isFetched: isTwabsFetched } = useUsersTotalTwab(usersAddress)
+
+  let usersTotalV4TwabBalanceUnformatted = twabs?.twab.amountUnformatted
+  let delegatedToAmount = ''
+
+  const queryResults = useUsersTicketDelegateAllPools()
+
+  const isFetched = isTwabsFetched && queryResults.every((queryResult) => queryResult.isFetched)
+
+  if (isFetched) {
+    queryResults.forEach((queryResult) => {
+      const result = queryResult.data
+      const delegatedToSelf = result?.[usersAddress] === usersAddress
+      if (delegatedToSelf) {
+        usersTotalV4TwabBalanceUnformatted = usersTotalV4TwabBalanceUnformatted.sub(
+          result?.ticketData.ticket
+        )
+      }
+    })
+
+    delegatedToAmount = formatUnits(usersTotalV4TwabBalanceUnformatted, 6)
```

Comment:
You can use `usePrizePoolTokens` and pull the decimals from there rather than hardcoding for USDC